.github/workflows/ci.yml

```
@@ -0,0 +1,157 @@
+    - name: Upload Trivy scan results to GitHub Security tab
+      uses: github/codeql-action/upload-sarif@v2
+      if: always()
+      with:
+        sarif_file: 'trivy-results.sarif'
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [test, build, lint, security]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)